examples/verizonmedia.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Verizon Ad Test</title>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<h2>Verizon Ad Test No Fallback</h2>
+<amp-ad
```

Comment:
Could you please move this example to `examples/amp-ad/ads.amp.html`?

Reply:
done and re-pushing code now